README.md

```diff
@@ -232,6 +233,26 @@ If it is not in the current working directory along with the indexer command we
 ~$ ./algorand-indexer daemon -c <full-file-location>/indexer.yml
 ```
 
+## Data Directory
+
+The Indexer data directory is the location where the Indexer can store and/or load data needed for runtime operation and configuration.  
+
+**It is a required argument for Indexer daemon operation. Supply it to the Indexer via the `--data-dir` flag.**
+
+
+### Auto-Loading Configuration
+
+The Indexer will scan the data directory at startup and load certain configuration files if they are present.  The files are as follows:
+
+- Indexer Configuration File
+  - Normally supplied with the `-c` flag, if the data directory contains a file named `indexer.yml` then the Indexer will assume it is an Indexer configuration file.
+- Parameter Enable/Disable Configuration File
+  - Normally supplied with the `--api-config-file` flag, if the data directory contains a file named `api_config.yml` then the Indexer will assume it is a Parameter Enable/Disable configuration file.
```

Comment:
I'm not sure saying "normally" conveys that this is the new recommended way to provide configuration files. Here is a suggestion, but maybe it is too terse. What do you think?
```suggestion
- `indexer.yml` - Indexer Configuration File
- `api_config.yml` - API Parameter Enable/Disable Configuration File
```